meta-client/src/main/java/org/apache/helix/metaclient/exception/MetaClientException.java

```diff
@@ -35,4 +35,114 @@ public MetaClientException(String message) {
   public MetaClientException(Throwable cause) {
     super(cause);
   }
+
+  public enum Code {
```

Comment:
Should we call ReturnCode ?? Code name sounds a bit confusing

Reply:
Thanks for the suggestion. Will change.